src/main/java/org/opentripplanner/middleware/utils/LatLongUtils.java

```diff
@@ -0,0 +1,105 @@
+package org.opentripplanner.middleware.utils;
+
+import java.util.Objects;
+import java.util.Random;
+
+/**
+ * Utility class to randomize lat/long coordinates. Information on this can be found here:
+ * https://gis.stackexchange.com/questions/25877/generating-random-locations-nearby. Original code (in Javascript) can
+ * be found and tested here: http://jsfiddle.net/hoolymama/56wzdtax/.
+ */
+public class LatLongUtils {
+    public static boolean IS_TEST = false;
```

Comment:
Again, I'm not too hot about this. I would create some form of randomiser interface or method that has two implementations: one for production and one for testing.

Reply:
Fair point. I have overloaded the method.